### PR TITLE
Fix new ClientWebSocketOptions tests on Windows 7

### PR DESCRIPTION
**Pull request by stephentoub**
cc: @davidsh

### CODE REVIEW

File: src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs [hunk 1]
```diff
@@ -8,12 +8,15 @@
...
 
 namespace System.Net.WebSockets.Client.Tests
 {
-    public class ClientWebSocketOptionsTests
+    public class ClientWebSocketOptionsTests : ClientWebSocketTestBase
     {
-        [Fact]
+        public ClientWebSocketOptionsTests(ITestOutputHelper output) : base(output) { }
+
+        [ConditionalFact(nameof(WebSocketsSupported)), MemberData(nameof(EchoServers))]
```
Comment (davidsh) [inline src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs]: Why include MemberData() if there are no parameters to the test method?

File: src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs [hunk 2]
```diff
@@ -24,7 +27,7 @@ public static void UseDefaultCredentials_Roundtrips()
             Assert.False(cws.Options.UseDefaultCredentials);
         }
 
-        [Fact]
+        [ConditionalFact(nameof(WebSocketsSupported)), MemberData(nameof(EchoServers))]
```
Comment (davidsh) [inline src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs]: Same as above comment.

File: src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs [hunk 3]
```diff
@@ -38,7 +41,7 @@ public static void SetBuffer_InvalidArgs_Throws()
             Assert.Throws<ArgumentOutOfRangeException>("buffer", () => cws.Options.SetBuffer(1, 1, new ArraySegment<byte>(new byte[0])));
         }
 
-        [Fact]
+        [ConditionalFact(nameof(WebSocketsSupported)), MemberData(nameof(EchoServers))]
```
Comment (davidsh) [inline src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs]: Same as above comment.

File: src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs (hunk 1 above)
Comment (stephentoub) [inline src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs]: Oops, copy and paste... didn't even see it was there :smile: Will remove.

**Comment by davidsh** [comment]
Test OuterLoop Windows 7 Debug

**Pull request merged**